astronomy/initial_state.proto.txt

```diff
@@ -0,0 +1,316 @@
+initial_state {
```

Comment:
The scripts that generate this file should be updated in a separate CL. They should also sort the bodies so that diffing is easier. #666.